### PR TITLE
subreddits added

### PR DESCRIPTION
**Pull request by abiaad**

### CODE REVIEW

File: templates/slacker.html [hunk 1]
```diff
@@ -41,6 +41,9 @@
 			<li><a href="https://www.reddit.com/r/nsfw_gifs">/r/nsfw_gifs</a></li>
 			<li><a href="https://www.reddit.com/r/Burstingout">/r/Burstingout</a></li>
 			<li><a href="https://www.reddit.com/r/bonermaterial">/r/bonermaterial</a></li>
+			<li><a href="https://www.reddit.com/r/facedownassup">/r/facedownassup</a></li>
+			<li><a href="https://www.reddit.com/r/pawg">/r/bonermaterial</a></li>
```
Comment (fartattorney) [inline templates/slacker.html]: link text should match up to the subreddit here.

**Comment by fartattorney** [comment]
good additions. fix the link text on `slacker.html` and we'll merge

**Comment by fartattorney** [comment]
thanks @abiaad. will go live this week

**Comment by abiaad** [comment]
I'll add more soon 😄

**Pull request merged**